# ActivateObstacle.srv
int64 obstacle_number
---
string response